### ISSUE TITLE
Experiments

### ISSUE THREAD

**Issue by Grutschus** [Grutschus/human-fall-detection#40]
This is a collection of experiments we want to run.
This is a follow up on our discussion from Friday #36.

- [x] #39 (vit-s_uniform-sampling_priority-labeling_paper-hyperparams)
- [x] vit-b_gaussian-sampling_priority-labeling_paper-hyperparams
- [x] vit-b_gaussian-sampling_priority-labeling_paper-hyperparams_weighted-ce-loss
- [ ] vit-b_uniform-sampling_priority-labeling_paper-hyperparams_undersampling-other-0.75
- [ ] vit-gaussian-sampling_priority-labeling_paper-hyperparams_undersampling-other-0.75
- [x] Repeat some promising experiments with the ViT-B K400 hyperparams from [here](https://github.com/OpenGVLab/VideoMAEv2/blob/master/scripts/finetune/vit_b_k400_ft.sh)

This list is not exhaustive. We'll add ideas and experiments as we progress, for example when #38 is done.
